README.md

```diff
@@ -30,10 +30,23 @@ See `scripts` section in package.json for other development commands. For
 example run client unit tests via `docker-compose exec bluetooth-client npm
 test`.
 
+### Configuration
+
+ | env variable | description
+----|----
+INITIAL_LATITUDE | latitude as float for the initial map position (WGS84)
+INITIAL_LONGITUDE | longitude as float for the initial map position
+INITIAL_ZOOM | mapbox zoom level, from 1 to 22
```

Comment:
Should this be added to the README, as well: MINIMUM_ZOOM seems to be missing from this list.